ironfish/src/rpc/routes/wallet/addTransaction.ts

```diff
@@ -14,6 +14,8 @@ export type AddTransactionRequest = {
 
 export type AddTransactionResponse = {
   accounts: string[]
+  hash: string
```

Comment:
We have a ticket for adding the hash to the response of `wallet/postTransaction`, too. If that endpoint returns the hash then the client will already have the hash before calling `wallet/addTransaction`.

I think it's fine to have the hash here too, though!